tee-worker/litentry/core/credentials/src/lib.rs

```diff
@@ -186,7 +186,7 @@ pub struct Proof {
 impl Proof {
 	pub fn new(sig: &Vec<u8>, issuer: &AccountId) -> Self {
 		Proof {
-			created_timestamp: now_as_millis(),
+			created: now_as_millis(),
```

Comment:
If the goal is to align with the spec, shouldn't `created` be in ISO-8601 format instead?

Reply:
Yes, we will change all datetimes representations from timestamps to ISO-8601

Reply:
Here or in a different PR?

Reply:
I'll add it here to make this change complete.